code/datums/quirks/negative.dm

```diff
@@ -138,6 +138,8 @@
 
 	if(ismoth(H) && prob(50))
 		heirloom_type = /obj/item/flashlight/lantern/heirloom_moth
+	else if(isfelinid(H) && prob(50))
+		heirloom_type = /obj/item/toy/cattoy
```

Comment:
Shouldn't this be a subtype like the moth lantern?

Reply:
Apparently that only exists to nerf light range, depressing

Reply:
and its still just a buff